src/settings/FinePolicy/components/EditSections/FinesSection/FinesSection.js

```diff
@@ -17,7 +17,6 @@ import OverdueFinesSectionColumn from '../RangeSection/OverdueFinesSectionColumn
 class FinesSection extends React.Component {
   static propTypes = {
     intl: PropTypes.object.isRequired,
```

Comment:
If there is only one prop, let's write it in one row

Reply:
Fixed